central/postgres/pruning.go

```diff
@@ -87,6 +87,9 @@ const (
 
 	// Delete the log imbues with old timestamp
 	pruneLogImbues = `DELETE FROM log_imbues WHERE timestamp < now() at time zone 'utc' - INTERVAL '%d MINUTES'`
+
+	// Delete orphaned PLOPs
+	pruneOrphanedPLOPs = `DELETE FROM listening_endpoints WHERE closetimestamp < now() at time zone 'utc' - INTERVAL '%d MINUTES'`
```

Comment:
Not sure that I follow that this is a orphaned plop? @JoukoVirtanen Do we not keep history about which plops were opened if the pod is still around?

Reply:
The only criteria for pruning is if the listening endpoint has been closed longer than some specified amount of time. We also need to add pruning for when the pod has been deleted, but the listening endpoint has not been reported as being closed. What criteria are you suggesting?

Reply:
I think improving the pruning criteria can be done in another PR.

Reply:
@JoukoVirtanen Fixing the PLOP criteria should probably be tracked with a ticket in addition to a PR.

Reply:
I have created a ticket here https://issues.redhat.com/browse/ROX-19833

There was already a ticket for not reporting listening endpoints that whose pods have been deleted https://issues.redhat.com/browse/ROX-19577

I have a draft PR here https://github.com/stackrox/stackrox/pull/7858

I don't think I am doing things the best way there yet. I need to write some thoughts down and do some testing.